pb_plugins/protoc_gen_dcsdk/docs.py

```diff
@@ -0,0 +1,97 @@
+# -*- coding: utf-8 -*-
+
+
+class Docs(object):
+    """
+    Docs
+
+    Based on https://github.com/protocolbuffers/protobuf/blob/a08b03d4c00a5793b88b494f672513f6ad46a681/src/google/protobuf/descriptor.proto
+    """
+
+    @staticmethod
+    def collect_docs(source_code_info):
+
+        docs = {}
+
+        docs['methods'] = []
+        docs['structs'] = []
+        docs['enums'] = []
+
+        for location in source_code_info.location:
+            if len(location.path) > 0:
+                path = location.path
+
+                if path[0] == 1:  # name
+                    pass
+                elif path[0] == 2:  # package
+                    pass
+                elif path[0] == 4:  # message
+                    struct_id = path[1]
```

Comment:
This would be more understandable if it branched out to methods.

Reply:
I agree that there is some refactoring to be done here, but I'd be happier if we could get this in, so that I can move on with the templates, and come back later to that code.

The main reason being that I wrote this while generating Python documentation, and I'm not yet completely sure if that's good enough for all our languages or not. One we have it working for a few languages, I will know better how to structure this.

Does that make sense to you?

Reply:
Sure!